rewards/df-volumedf.md

```diff
@@ -22,13 +22,13 @@ Data Farming rewards farmers that allocate their veOCEAN tokens to assets that *
 Thus, if you really want to max out Volume DF APY:
 1. Lock your OCEAN for 4 years to receive 100% voting power.
 1. Have other participants delegate their Allocation Power to you.
-1. Create & publish assets (and make $ in selling them) — or work with people who can.
+1. Create & publish assets (and make \$ in selling them) — or work with people who can.
```

Comment:
The is no need for the extra backslash. It's actually rendered in the UI

Reply:
Hmm, it's needed inside VS Code as it's a special character. Will review in docs.